README.md

```diff
@@ -252,20 +252,24 @@ The following generic methods are added to `nemo.view`
 
 `@returns {Promise}` resolves to true or rejected
 
-#### _wait(locatorString[, timeout])
+#### _wait(locatorString[, timeout, msg])
```

Comment:
I didn't understand use of [] to be honest. My guess was whatever that is optional goes into []. Now I understood. will fix it now